### ISSUE TITLE
Font Size and Export should be focusable

### ISSUE THREAD

**Issue by sehilyi** [gosling-lang/gosling.js#1051]
Currently, changing the font size or exporting a visualization using a keyboard is impossible. The following screenshot shows that those buttons are not accessible (i.e., the absence of yellow labels).

![Screenshot 2024-04-11 at 12 04 37 PM](https://github.com/gosling-lang/gosling.js/assets/9922882/be96159f-bd49-4e1c-ba1f-798b69d9402f)

Thanks to @LDubya!
